Sources/TensorFlow/Core/ShapedArray.swift

```diff
@@ -703,6 +703,7 @@ extension ShapedArray: ExpressibleByArrayLiteral where Scalar: TensorFlowScalar
     public typealias ArrayLiteralElement = _TensorElementLiteral<Scalar>
     @inlinable
     public init(arrayLiteral elements: _TensorElementLiteral<Scalar>...) {
+        precondition(!elements.isEmpty, "Cannot create a 'ShapedArray' with no elements")
```

Comment:
Should we end the error messages with "." (i.e., periods) so they form full sentences?

Reply:
Good suggestion.  Done.